cypress/e2e/feeder-rss-import.cy.ts

```diff
@@ -0,0 +1,47 @@
+describe("Feeder RSS Import", () => {
+  before(() => {
+    cy.login(Cypress.env("TEST_PRX_USER"), Cypress.env("TEST_PRX_PASS"));
+    Cypress.config({ baseUrl: `https://${Cypress.env("FEEDER_HOST")}` });
+  });
+
+  it("imports an RSS feed", () => {
+    const now = new Date().toISOString();
+    const canary = `Acceptance Test: ${now}`;
+
+    cy.visit("/podcasts");
+    cy.contains("My Podcasts");
+
+    // create a new podcast
+    cy.get('a.btn-success[href="/podcasts/new"]').click();
+    cy.contains("New Podcast");
+    cy.get("#podcast_title").type(canary);
+    cy.get("#podcast_default_feed_attributes_itunes_category").select("Arts", { force: true });
+    cy.contains(".btn", "Create").click();
+    cy.contains("Podcast created");
+
+    // start importing a url
+    cy.contains("a", "Import").click();
+    cy.get("#podcast_import_url").type("https://ryan-cdn.prxu.org/test-rss-import.xml");
```

Comment:
Seems like this ought not to be in a developer/ryan specific bucket - I believe we have test files in some other buckets....like here: `https://prx-tech.s3.amazonaws.com/test/feed/feed-rss.xml`